client/src/components/InternshipTag.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Box, useTheme } from "@mui/material";
+
+const InternshipTag = (props) => {
+  let theme = useTheme();
+
+  return (
+    <>
```

Comment:
If we're returning a single element `Box` here, the fragment is unnecessary and we can remove it.